src/kakarot/model.cairo

```diff
@@ -7,18 +7,36 @@ from starkware.cairo.common.uint256 import Uint256
 from starkware.cairo.common.dict import DictAccess
 
 namespace model {
+
+    // @dev Stack with a 1024 items size. Each item is a 256 bits word. The stack is used by most 
+    // @dev opcodes to consume their parameters from.
+    // @dev Each element is represented by two 128bits (16bytes) chunks.
+    // @param word_dict_start - pointer to a DictAccess array used to simulate a read/write memory
+    // @param word_dict - pointer to the end of the DictAccess array
+    // @param len_16_bytes - length of the DictAccess array
     struct Stack {
         word_dict_start: DictAccess*,
         word_dict: DictAccess*,
         len_16bytes: felt,
     }
 
+    // @dev Transient memory maintened by the EVM during an execution which doesn't persist
```

Comment:
save comment as above for @notice and https://www.evm.codes/about#memory

also, you should install a spell check in your IDE as you have ! (maintened=>maintained)